ecoscore/data/fr_packaging_shapes.csv

```diff
@@ -28,8 +28,9 @@ Film de regroupement,"0,1"
 Opercule,"0,1"
 Bâtonnet,"0,1"
 Attache,"0,1"
+# Lines below are not in the Eco-Score file, but they are used for many products
+# so we assign the ratio for the closest component
 Vrac,0
-Conteneur,1
```

Comment:
It's because "conteneur" is a synonym in the packaging_recycling taxonomy, so that "conteneur à verre" matches "benne à verre", "poubelle à verre" etc.

Reply:
oh yes, we could see it in the sentences "1 carton à recycler (conteneur papier)"